Sources/StreamVideo/Call.swift

```diff
@@ -928,8 +928,8 @@ public class Call: @unchecked Sendable, WSEventsSubscriber {
     /// will be thrown to indicate that the operation is not supported.
     ///
     /// - Warning: A maximum of one output of each type may be added.
-    public func addCapturePhotoOutput(_ capturePhotoOutput: AVCapturePhotoOutput) throws {
-        try callController.addCapturePhotoOutput(capturePhotoOutput)
+    public func addCapturePhotoOutput(_ capturePhotoOutput: AVCapturePhotoOutput) async throws {
+        try await callController.addCapturePhotoOutput(capturePhotoOutput)
```

Comment:
Even though we could simply wrap the call internally to a Task, and that will help us by not needing to add the async. At the same time it would invalidate our API as we aren't able then to throw any error that occurred (as that requires waiting for the task to execute).

So that's as small as possible while remaining aligned with the needs of the method.

Reply:
ok, let's mention this in the changelog - best in this PR so we don't forget.

Reply:
I don't want to make any other changes in this PR to avoid breaking it. In another PR is alright also because we are not targeting develop.